modules/nw-egressnetworkpolicy-about.adoc

```diff
@@ -82,7 +82,12 @@ The egress firewall policy rules are evaluated in the order that they are define
 
 If you use DNS names in any of your egress firewall policy rules, proper resolution of the domain names is subject to the following restrictions:
 
-* Domain name updates are polled based on the TTL (time to live) value of the domain returned by the local non-authoritative servers.
+ifdef::openshift-sdn[]
+* Domain name updates are polled based on a time-to-live (TTL) duration. By default, the duration is 30 seconds. When the egress firewall controller queries the local name servers for a domain name, if the response includes a TTL that is less than 30 seconds, the controller sets the duration to the returned value. If the TTL in the response is greater than 30 minutes, the controller sets the duration to 30 minutes. If the TTL is between 30 seconds and 30 minutes, the controller ignores the value and sets the duration to 30 seconds.
+endif::openshift-sdn[]
+ifdef::ovn[]
+* Domain name updates are polled based on a time-to-live (TTL) duration. By default, the duration is 30 minutes. When the egress firewall controller queries the local name servers for a domain name, if the response includes a TTL, the controller sets the duration to the returned value.
+endif::ovn[]
```

Comment:
for the ovn section this is not exactly true we do not change the ttl the minimum wakeup time for the process is 30 minutes the egressfirewall always queries the dnsName with the smallest ttl and if all the dnsNames have a ttl of more then 30 minutes it will query the one with the smallest ttl every 30 minutes. I might be able to make this more clear over slack if needed

Reply:
@JacobTanenbaum, I revised after the quick chat.  Thanks and please check again.